README.md

```diff
@@ -32,11 +32,14 @@ Follow these steps to build an AMI for GraphDB using Packer:
    instance type, VPC ID, and subnet ID. To do so, create a variables file `variables.pkrvars.hcl`, example file: 
 
    ```bash
-   gdb_version  = "10.3.2"
+   gdb_version  = "10.3.3"
    build_aws_regions = ["eu-central-1"]
-   build_instance_type = "m5.large"
    build_vpc_id = "<your-vpc-id>"
    build_subnet_id = "<your-subnet-id>"
+   build_instance_type_x86-64    = "m5.large"
+   build_instance_type_arm64     = "t4g.small"
+   source_ami_name_filter_arm64  = "ubuntu/images/hvm-ssd/ubuntu-*-22.04-arm64-server-*"
+   source_ami_name_filter_x86-64 = "ubuntu/images/hvm-ssd/ubuntu-jammy-22.04-amd64-server-*"
```

Comment:
Do we have to specify the codename if the version is given? Idk so that's why I'm asking

Reply:
Removed